### PR TITLE
[RHELC-399] Add test description

### PR DESCRIPTION
**Pull request by hosekadam**
* add description and links for config-file, changed-yum-conf, one-kernel-scenario
* change of wrong number of tier in config-file main.fmf file

Jira Issue: [RHELC-399](https://issues.redhat.com/browse/RHELC-399)

Checklist
- [ ] PR meets acceptance criteria specified in the Jira issue
- [ ] PR has been tested manually in a VM (either author or reviewer)
- [ ] Jira issue has been made public if possible
- [ ] `[RHELC-]` is part of the PR title 
- [x] Code and tests are documented properly
- [x] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending`

### CODE REVIEW

File: tests/integration/tier1/config-file/main.fmf [hunk 1]
```diff
@@ -1,5 +1,15 @@
-summary: config file
+summary: Test config file
 
-tier: 0
+description: |
+    Testing the configuration file option. There is need to test the
+    priority of the possible configuration files paths. Combined with
```
Comment (kokesak) [inline tests/integration/tier1/config-file/main.fmf:5]:
```suggestion
    priority of the possible configuration files paths combined with
```

File: tests/integration/tier1/config-file/main.fmf [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+    the cli options and check if right data were loaded and if needed
```
Comment (kokesak) [inline tests/integration/tier1/config-file/main.fmf:6]:
```suggestion
    the CLI options. Verify that the correct data were loaded and if applicable
```

File: tests/integration/tier1/changed-yum-conf/main.fmf [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+
+description: |
+    Test the scenario in which the user modifies /etc/yum.conf
+    before the conversion. In that case during the conversion
+    the config file does not get replaced with the config file
+    from the RHEL package (%config(noreplace)) and we need to
+    make sure that we patch the config file to get rid of the
+    distroverpkg config key. Leaving the distroverpkg key there
+    would lead to errors when calling yum after the conversion for not
+    expanding the $releasever variable properly.
```
Comment (kokesak) [inline tests/integration/tier1/changed-yum-conf/main.fmf:11]:
```suggestion
    Test the scenario in which the user modifies /etc/yum.conf
    before the conversion. In that case, during the conversion,
    the config file does not get replaced with the config file
    from the RHEL package. Make sure that C2R patch the config file
    and verify that the yum works after a successful conversion. 
```

Comment (kokesak) [reply tests/integration/tier1/changed-yum-conf/main.fmf:11]: Keep it simple and describe only what is happening in the test. The details of implementation should not be present here.

**Review by kokesak — CHANGES_REQUESTED** [review]

File: tests/integration/tier1/config-file/main.fmf [hunk 4]
```diff
@@ -1,5 +1,15 @@
-summary: config file
+summary: Test config file
 
-tier: 0
+description: |
+    Testing the configuration file option. There is need to test the
+    priority of the possible configuration files paths combined with
+    the CLI options. Verify that the correct data were loaded and if applicable
+    a warning is printed.
```
Comment (r0x0d) [inline tests/integration/tier1/config-file/main.fmf:7]:
```suggestion
    Run Convert2RHEL using a configuration file for sensitive information. 
    Verify that the conversion can be done with different configuration paths, as well, CLI options and to assert that the configuration will take precendence over the CLI.
    Verify that the correct data were loaded and if applicable a warning is printed.
```

**Review by r0x0d — CHANGES_REQUESTED** [review]
Just one small change to make the description easier to read

File: tests/integration/tier1/config-file/main.fmf (hunk 4 above)
Comment (hosekadam) [reply tests/integration/tier1/config-file/main.fmf:7]: Thanks @r0x0d I've missed this change, sorry. Updated and rebased

**Review by r0x0d — APPROVED** [review]

**Review by bocekm — APPROVED** [review]

**Pull request merged**